package.json

```diff
@@ -22,13 +22,15 @@
     "start": "yarn build:watch",
     "lint": "yarn workspaces foreach --all --parallel run lint",
     "prettier": "prettier",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "publish-all": "yarn workspaces foreach --all --topological-dev npm publish"
```

Comment:
Just to double-check -- does this command respect the public/private aspect of each package? I'm assuming it does and just reads each package's package.json to figure that out.

Reply:
Good question! It does indeed!